WordPress/Classes/Models/WPAccount.m

```diff
@@ -44,8 +48,8 @@ - (void)didTurnIntoFault
 {
     [super didTurnIntoFault];
     
-    _restApi = nil;
-    _xmlrpcApi = nil;
+    self.restApi = nil;
+    self.xmlrpcApi = nil;
```

Comment:
Same as in Blog.m. By hitting the setter we _might_ be actually instantiating both, the restAPI and the xmlrpcAPI